src/common/utils/untildify.ts

```diff
@@ -0,0 +1,12 @@
+import os from 'os';
+
+const homeDir = os.homedir();
+
+/***
+ * @description: Replaces '~' at the start of the path and following '/' or '\' with home
+ * @param {string} path
+ * @return {string} path
+ */
+export const untildify = (path: string): string => {
+  return homeDir ? path.replace(/^~(?=$|\/|\\)/, homeDir) : path;
```

Comment:
Will this work on Windows?

Reply:
It will work on every OS that has `os.homedir()` set correctly.